pandas/tests/io/test_sql.py

```diff
@@ -69,6 +79,17 @@
 except ImportError:
     SQLALCHEMY_INSTALLED = False
 
+try:
+    from sqlalchemy.ext.asyncio import (
+        create_async_engine,
+        AsyncConnection,
+        AsyncSession
+    )
+    from sqlalchemy import text
```

Comment:
is this only in a certain version of sqlalchemy?

Reply:
This is for SQLAlchemy 1.4 and higher.